browser/src/Services/Browser/index.tsx

```diff
@@ -164,7 +178,7 @@ export const activate = (
         const activeBuffer = editorManager.activeEditor.activeBuffer
 
         const browserLayer = activeLayers[activeBuffer.id]
-        if (browserLayer) {
+        if (browserLayer && !browserLayer.inputInProgress) {
```

Comment:
I think we might want to refine this logic a bit. Semantically, it would probably make more sense to be in the `enabled` filter, since this is gating whether or not the command is needed (vs the `execute`, which  should really just be responsible for _executing_ the command).

I don't think this would handle both cases of having a textarea / input in focus and the address bar. I think a slightly more robust condition would be to only allow our scroll commands if the webview is focused _and_ we're not in an input / textarea element (this is what I implemented in #2312 ) - let me know what you think though!